### PR TITLE
add ! name and description to DeduplicationSet

### PR DESCRIPTION
**Pull request by vitali-yanushchyk-valor**
del ! error from DeduplicationSet
chg ! admin panel

### CODE REVIEW

File: src/hope_dedup_engine/apps/api/admin/deduplicationset.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        ("state", ChoicesFieldComboFilter),
+        ("created_at", DateRangeFilter),
+        ("updated_at", DateRangeFilter),
+        DjangoLookupFilter,
+    )
+
+    def short_name(self, obj):
+        if obj.name:
+            return (
+                (obj.name[:MAX_STR_DISPLAY_LENGTH] + "...")
```
Comment (domdinicola) [inline src/hope_dedup_engine/apps/api/admin/deduplicationset.py]: let's remove this truncates

File: src/hope_dedup_engine/apps/api/models/deduplication.py [hunk 2]
```diff
@@ -50,6 +56,18 @@ class State(models.IntegerChoices):
     updated_at = models.DateTimeField(auto_now=True)
     notification_url = models.CharField(max_length=255, null=True, blank=True)
 
+    def __str__(self) -> str:
+        def shorten(text: str, trim: int = MAX_STR_DISPLAY_LENGTH) -> str:
+            return (
+                (text[:trim] + "...")
+                if text and len(text) > trim
```
Comment (domdinicola) [inline src/hope_dedup_engine/apps/api/models/deduplication.py]: let's remove the trim

File: src/hope_dedup_engine/apps/api/models/deduplication.py [hunk 3]
```diff
@@ -50,6 +56,18 @@ class State(models.IntegerChoices):
...
+                else text or EMPTY_PLACEHOLDER
+            )
+
+        if not self.name and not self.description:
```
Comment (domdinicola) [inline src/hope_dedup_engine/apps/api/models/deduplication.py]: let's ignore description

File: src/hope_dedup_engine/apps/api/models/deduplication.py [hunk 4]
```diff
@@ -1,12 +1,12 @@
...
+from typing import Any, Final, override
 from uuid import uuid4
 
 from django.conf import settings
 from django.db import models
 
 from hope_dedup_engine.apps.security.models import ExternalSystem
 
-REFERENCE_PK_LENGTH = 100
+REFERENCE_PK_LENGTH: Final[int] = 100
```
Comment (domdinicola) [inline src/hope_dedup_engine/apps/api/models/deduplication.py:9]: is this still used? @vitali-yanushchyk-valor

Comment (vitali-yanushchyk-valor) [reply src/hope_dedup_engine/apps/api/models/deduplication.py:9]: > is this still used? @vitali-yanushchyk-valor

Yes - it's other part of code. @domdinicola

**Review by domdinicola — APPROVED** [review]

**Pull request merged**